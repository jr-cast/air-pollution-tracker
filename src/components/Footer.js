const Footer = () => (
  <div className="footerCont">
    Copyright &copy; | Jose R. Casta√±os
  </div>
);

export default Footer;
