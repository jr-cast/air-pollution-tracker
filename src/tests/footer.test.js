import { screen, render } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Footer from '../components/Footer';

describe('Testing the Footer component', () => {
  test('Footer component renders properly', () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('When Footer gets rendered inner text is displayed correctly', () => {
    render(<Footer />);
    screen.debug();
    expect(screen.getByText('Copyright © | Jose R. Castaños')).toBeTruthy();
  });
});
