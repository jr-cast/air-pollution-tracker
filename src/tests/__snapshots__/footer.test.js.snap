// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders properly 1`] = `
<div
  className="footerCont"
>
  Copyright © | Jose R. Castaños
</div>
`;

exports[`Testing the Footer component Footer component renders properly 1`] = `
<div
  className="footerCont"
>
  Copyright © | Jose R. Castaños
</div>
`;
